schema/pgsql/upgrades/031.sql

```diff
@@ -0,0 +1,83 @@
+ALTER TABLE channel
+    ADD COLUMN changed_at bigint NOT NULL DEFAULT EXTRACT(EPOCH FROM NOW()) * 1000,
+    ADD COLUMN deleted boolenum NOT NULL DEFAULT 'n';
+
+CREATE INDEX idx_channel_changed_at ON channel(changed_at);
+
+ALTER TABLE contact
+    ADD COLUMN changed_at bigint NOT NULL DEFAULT EXTRACT(EPOCH FROM NOW()) * 1000,
+    ADD COLUMN deleted boolenum NOT NULL DEFAULT 'n';
+
+CREATE INDEX idx_contact_changed_at ON contact(changed_at);
+
+ALTER TABLE contact_address
+    ADD COLUMN changed_at bigint NOT NULL DEFAULT EXTRACT(EPOCH FROM NOW()) * 1000,
+    ADD COLUMN deleted boolenum NOT NULL DEFAULT 'n';
+
+CREATE INDEX idx_contact_address_changed_at ON contact_address(changed_at);
+
+ALTER TABLE contactgroup
+    ADD COLUMN changed_at bigint NOT NULL DEFAULT EXTRACT(EPOCH FROM NOW()) * 1000,
+    ADD COLUMN deleted boolenum NOT NULL DEFAULT 'n';
+
+CREATE INDEX idx_contactgroup_changed_at ON contactgroup(changed_at);
+
+ALTER TABLE contacgroup_member
+    ADD COLUMN changed_at bigint NOT NULL DEFAULT EXTRACT(EPOCH FROM NOW()) * 1000,
+    ADD COLUMN deleted boolenum NOT NULL DEFAULT 'n';
+
+CREATE INDEX idx_contacgroup_member_changed_at ON contacgroup_member(changed_at);
```

Comment:
You have a typo here, `contacgroup_member` -> `contactgroup_member`.

Reply:
Oops. Thanks.